framework/include/meshgenerators/MeshDiagnosticsGenerator.h

```diff
@@ -32,9 +32,13 @@ class MeshDiagnosticsGenerator : public MeshGenerator
   /// Routine to check sideset orientation near subdomains
   void checkSidesetsOrientation(const std::unique_ptr<MeshBase> & mesh) const;
   //// Routine to check is mesh is fully covered in sidesets
-  void checkWaterTightSidesets(const std::unique_ptr<MeshBase> & mesh) const;
+  void checkWatertightSidesets(const std::unique_ptr<MeshBase> & mesh) const;
   //// Routine to check is mesh is fully covered in nodesets
   void checkWatertightNodesets(const std::unique_ptr<MeshBase> & mesh) const;
+  /// Helper function that finds the intersection of the _watertight_boundaries vector and the provided one
```

Comment:
it does not use _waterright_boundaries, but rather a copy that is sorted.
so just write the description in a more general manner that does not mention _watertight_boundaries